src/DIRAC/Core/Celery/Task/OptimizeTask.py

```diff
@@ -0,0 +1,8 @@
+from DIRAC.Core.Celery.CeleryApp import celeryApp
+from DIRAC.WorkloadManagementSystem.Client.JobState.JobState import JobState
+from DIRAC.WorkloadManagementSystem.OptimizerAdministrator.OptimizerAdministrator import OptimizerAdministrator
+
+
+@celeryApp.task
+def optimize(jobState: JobState):
+    OptimizerAdministrator().optimize(jobState)
```

Comment:
The result of `optimize` should be treated.

Reply:
Well, we definitely can, but I don't know what we should do with the result. I think all the business logic and logging in case of errors should be kept in the OptimizerAdminstrator class.